hawkular-command-gateway/hawkular-command-gateway-itest/src/test/java/org/hawkular/cmdgw/ws/test/AbstractCommandITest.java

```diff
@@ -39,6 +39,7 @@
         int httpPort = portOffset + 8080;
         baseGwUri = "ws://" + host + ":" + httpPort + "/hawkular/command-gateway";
         authentication = "{\"username\":\"" + testUser + "\",\"password\":\"" + testPasword + "\"}";
+        // System.out.println("testPasword = "+ testPasword);
```

Comment:
Ok, then just leave a friendly comment above it. Otherwise someone will eventually just remove it :)